src/Engine/Component/SoundComponents/SoundComponent.cpp

```diff
@@ -0,0 +1,12 @@
+#include "SoundComponent.hpp"
+
+SoundComponent::SoundComponent()
+{
+}
+
+#ifdef HAVE_FMOD_LIBRARY
```

Comment:
Does cmake define this or do we need to do it manually through visual studio?

Reply:
It's a #define in SoundSystem.hpp that you have to manually change.